Dheavyman/More-Recipes
# #153979135 user profile page

#### What does this PR do?
Have a user profile page on the application
#### Description of Task to be completed?
User should be able to view his/her profile with the details of users activities
#### How should this be manually tested?
- Clone the repository and change directory into it
- Install the dependencies with the command `npm install`
- Start the server with the command `npm run start:dev`
- Visit the app on the browser with this url `127.0.0.1:3000/8080`
- Create an account and login to the application
- Visit the dashboard page to see the user profile details
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#153979135
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
